packages/vuetify/src/components/VField/VField.sass

```diff
@@ -399,7 +401,13 @@
       &__notch
         flex: none
         position: relative
-        max-width: calc(100% - var(--v-input-control-height))
+        max-width: calc(100% - $field-control-affixed-padding * 2)
+
+        @at-root
+          #{selector.append('.v-field--rounded', &)},
+          #{selector.append('[class^="rounded-"]', &)},
+          #{selector.append('[class*=" rounded-"]', &)}
```

Comment:
I understand why this is needed but (I believe) this would the first code of its type in the framework. Would love to get thoughts from anyone else on the @vuetifyjs/core-team

Reply:
These 3 lines (with `selector.append`) are the exact copy from a few lines above. I wish I would not do it, but adding `--v-field-border-radius` feels like it would need to go to `dev` branch first and I want to patch regression, so it needs to go to `master`